dataactvalidator/config/sqlrules/fabs43_detached_award_financial_assistance_2.sql

```diff
@@ -8,6 +9,7 @@ SELECT
 FROM detached_award_financial_assistance
 WHERE submission_id = {0}
     AND (COALESCE(place_of_performance_zip4a, '') = ''
-         OR place_of_performance_zip4a = 'city-wide')
+         OR place_of_performance_zip4a = 'city-wide'
+         OR place_of_performance_zip4a ~ '^\d\d\d\d\d$')
```

Comment:
It might be simpler to do one test here of `COALESCE(place_of_performance_zip4a, '') !~ '^\d\d\d\d\d\-?\d\d\d\d$'`